### DISCUSSION TITLE
Regular 'What's new in Athens?' updates

### DISCUSSION THREAD

**Discussion by joelhans** [athensresearch/athens#1359] [Operations]
For those who haven't seen it, @sawhney17 created an excellent page for **What's new in Athens** for the month of May: https://www.notion.so/What-s-new-in-Athens-da66d490d9f84abb8bf4f0732362be07

The Discord community loved it, so we want to make it a regular thing, and want to get some community feedback on what type of update would be most useful or interesting to read.

This is related to both #1355 and #1357, but I see these updates as a unique piece of content that would eventually be mentioned in a newsletter, but wouldn't necessarily be the foundation for it.

## Current state

- Most users of an open-source project want to know what's going on!
- Athens Research is not doing a great job of ☝️ right now.
- The only ways to hear about updates to the Athens app is through the `#announcements` channel in our [Discord server](https://discord.gg/as9h8yHNfD) or the [releases](https://github.com/athensresearch/athens/releases) in this repo.
  - Maybe an occasional Tweet.

## Questions

- Who is going to create this on a regular basis? 
  - @sawhney17?
  - Regular contribution is phenomenal, but we also want the process to be transparent so anyone could step in and create a future update.
- What kind of content should we add to these updates?
  - Product updates, aka commits to the the `athensresearch/athens` repo
  - Community highlight/spotlight news
  - Core team news
- How often should we create this content?
- Where should we publish these updates initially? Where is the "source of truth"?
  - The [Athens handbook](https://athensresearch.gitbook.io/handbook/)
  - GitHub discussions
  - The [Athens blog](https://athens-research.ghost.io)
  - An email newsletter (#1355)

## Next steps

- [ ] Collect community feedback.
- [ ] Create a documented process in the comments below for who, when, and how this is published.
- [ ] Roll out an update before the end of June!

**Comment by sawhney17** [comment]
I most certainly don't mind writing, but I agree that it is important to discuss about the transparency thing. 

I think a monthly update seems like a good idea. With this we will actually have something solid to put in it. For instance, we are at about the half way point of the month. And so far the biggest merged feature has been shift click to open pages in the sidebar via search. But there are quite a few WIP PRs that are coming soon which are pretty cool and will be pushed to the product in the second half of the month making it kind of uneven in a way. Same applies to the other things that are going to be on the doc development is not linear. The shorter the time frame, the more it _will_ deviate from the mean. 

I think the Ghost blog is a good idea for a location to put it on, but we shouldn't saturate the ghost blog with just updates we should probably create more content which is another topic for discussion. The ulysses blog is a good example with feature spotlights, workflow spotlights and updates as well. The handbook, To me, seems like more of an evergreen and not so much _time_ dependent. 

Also, the GitHub discussions are really unwieldily, at least for me and probably a lot of the less technically minded folks. I think the end goal should be migrating to a dedicated forum similar to something like the [Logseq Forum](forums.logseq.com) although this is probably a completely separate topic of discussion. 

I think what you mentioned about the topics sounds good. Perhaps we could do smaller updates on the emails as well on these topics as well.
